### PR TITLE
Added C64/C128 SuperCPU accelerator functions and started on a generic framework for accelerators.

### PR DESCRIPTION
**Pull request by blackystardust**
Added C64/C128 SuperCPU accelerator functions and started on a generic framework for accelerators.

### CODE REVIEW

**Comment by blackystardust** [comment]
Added some accelerator test code to test and show the operation of the accelerators.

**Comment by oliverschmidt** [comment]
Is this asking to be merged in the current state or is this rather work-in-progress shared to get feedback on the design before investing further?

File: asminc/accelerator.inc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+;
+; Accelerator definitions.
+;
+
+; ---------------------------------------------------------------------------
+; Speed definitions for all accelerator, to be used as input for the 'set'
+; functions.
+
+
+SPEED_SLOW := $00
```
Comment (oliverschmidt) [inline asminc/accelerator.inc]: ":=" is intended for addresses while "=" is intended for general constants. See http://cc65.github.io/doc/ca65.html#ss6.1

Comment (blackystardust) [reply asminc/accelerator.inc]: Any comments on the design are welcome of course, but besides the change needed for the constant definition it is ready for merging, other accelerator code will follow after it gets merged, including for other platforms like apple2 and pet.

**Comment by blackystardust** [comment]
Fixed the non-address constants in my latest commit.

**Comment by oliverschmidt** [comment]
> Any comments on the design are welcome of course, but besides the change needed for the constant definition it is ready for merging,

Thanks for the feedback. I see - and as no one complained so far I'm going to merge now...

> other accelerator code will follow after it gets merged, including for other platforms like apple2 and pet.

Ah, I wasn't aware that you're into the Apple2 too...

**Pull request merged**